src/api.js

```diff
@@ -62,10 +62,15 @@ api.declare({
     '**Warning** this api end-point is **not stable**.',
   ].join('\n'),
 }, async function(req, res) {
-  res.reply(
-    _.map(
+  res.reply( 
+    _.map( 
       await this.rabbit.exchanges(),
-      elem => elem.name
+      elem => JSON.stringify(
```

Comment:
It looks like this is returning a JSON string embedded in the overall JSON response?